src/vs/workbench/contrib/chat/browser/actions/chatCodeblockActions.ts

```diff
@@ -93,7 +95,11 @@ export function registerChatCodeBlockActions() {
 		}
 
 		run(accessor: ServicesAccessor, ...args: any[]) {
```

Comment:
When does this command run in the accessibility view? If the user has a keybinding for it? I wonder whether this should be a separate command, since it seems like copying a codeblock from the accessibility view is a different job from copying one from the chat view, and it looks like we aren't trying to get usage telemetry from it anyway.

Reply:
That seems fine to do for the copy action. The others are more involved so I don't want to copy all of that code